.github/workflows/codequality.yml

```diff
@@ -24,7 +17,7 @@ jobs:
         uses: actions-rs/toolchain@v1
         with:
           profile: minimal
```

Comment:
would like to follow some best-practices (normally you're running your whole CI for all channels just in case), can we run clippy for MSVC and stable if a regression causes clippy to be insensible to bad code